demo/src/app/components/accordion/accordion.component.html

```diff
@@ -1,8 +1,8 @@
-<ngbd-component-docs [name]="accordionDocs.name"
+<ngbd-component-docs [name]="accordionDocs.selector"
```

Comment:
We should have 2 separate things:
* name (class name) => people need to list it in `[directives]`
* selector